### PR TITLE
Send emails using smtplib directly (without pyramid mailer).

### PR DESCRIPTION
**Pull request by arnaud-morvan**
Also note the email template files openned using universal newline support.

This fix error 550 "Bare LF found" with free.fr.

Relate #700

### CODE REVIEW

**Comment by arnaud-morvan** [comment]
@asaunier: I will put that branch on demo for testing after travis finished to build the docker image.

**Comment by julsbreakdown** [comment]
:1st_place_medal:

**Comment by arnaud-morvan** [comment]
@asaunier @gberaudo : Can someone have a look, this pass Travis now and is testable on demov6.

**Comment by asaunier** [comment]
@stef74 @desnoes Could you have the demo tested to make sure emails sent by the demo server are actually received by any kind of email addresses? Thanks!

File: c2corg_api/emails/email_service.py [hunk 1]
```diff
@@ -20,8 +23,9 @@ def _get_file_content(self, lang, key):
         filepath = os.path.dirname(__file__) + '/i18n/%s/%s' % (lang, key)
         if not os.path.isfile(filepath):
             filepath = os.path.dirname(__file__) + '/i18n/%s/%s' % ('fr', key)
-        f = open(filepath, 'r')
-        return f.read().rstrip()  # No trailing new line in email subject!
+        with open(filepath, 'rU') as f:  # Use universal newline support.
+            content = f.read().rstrip()  # No trailing new line in subject!
+        return content
```
Comment (asaunier) [inline c2corg_api/emails/email_service.py]: Don't you need to move the ``return`` statement to the ``with``block where ``content`` is defined?

**Comment by desnoes** [comment]
@asaunier: i will do some tests today.

File: c2corg_api/emails/email_service.py (hunk 1 above)
Comment (arnaud-morvan) [reply c2corg_api/emails/email_service.py]: Yes it should be better for the case the file does not exist.

**Comment by arnaud-morvan** [comment]
I've tried to send a password recover email to an address in free.fr
The new error is 550 spam detected.

**Comment by arnaud-morvan** [comment]
I sent an email to proxad.net as explained in http://postmaster.free.fr/#faux_positifs, waiting for a response now ...

**Comment by arnaud-morvan** [comment]
Seems that Free SPAM policy has changed, I've just received a recover password email from demov6 in a free.fr mailbox.

**Comment by desnoes** [comment]
@arnaud-morvan : i have also tested and receive the email with my yahoo, gmail and free adresses :). Thanks. I have asked people from the association to do more tests with other email adresses. I would like to check it works with hotmail. We had some problem with this mail in the past.

**Comment by asaunier** [comment]
It seems Free has changed things about the emails we sent and now emails sent by the demo are received as expected. Is this PR needed as well?

**Comment by arnaud-morvan** [comment]
@asaunier : Note that this does not fix the "550 spam detected" but "550 bare LF found"

**Comment by asaunier** [comment]
> Note that this does not fix the "550 spam detected" but "550 bare LF found"

@arnaud-morvan OK, thanks for explaining :)

> Seems we should merge this PR now, if no one emits comments about it, I will merge today.

LGTM!

**Pull request merged**